upstream.go

```diff
@@ -1296,9 +1296,10 @@ func (uc *upstreamConn) handleMessage(ctx context.Context, msg *irc.Message) err
 		if err := parseMessageParams(msg, &name); err != nil {
 			return err
 		}
-		ch, err := uc.getChannel(name)
-		if err != nil {
-			return err
+		ch := uc.channels.Get(name)
+		if ch == nil {
+			uc.forwardMsgByID(ctx, downstreamID, msg)
+			return nil
```

Comment:
Hm, actually, this message is sent when the topic changes. When the client sends a `TOPIC` command to request the topic, I think `RPL_TOPIC` is used?